modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/containers/ActivationKeysTable/components/Header/hooks/useFilters.js

```diff
@@ -196,32 +196,26 @@ export default function useFilters(setFilterTerm, productName) {
 			) {
 				if (
 					!isNaN(filters.keyType.value.hasOnPremise) &&
-					filters.keyType.value.hasOnPremise
+					filters.keyType.value.hasOnPremise &&
```

Comment:
`!isNaN(filters.keyType.value.hasOnPremise)` verify if the value is not null or undefined. But those values are [falsy](https://developer.mozilla.org/en-US/docs/Glossary/Falsy), so I guess that you can remove this validation (`!isNaN..`)